### PR TITLE
Make sure platform and lane are correctly assigned on edit

### LINKED ISSUE #797: Storing the platform + lane as part of a project update/edit isn't working
Using the old "frontend" at least, my `Project` object looks like this
```
 @lane="ios",
 @platform="test",
```

it should be the opposite. I haven't tried with the new front-end - but we have to test this behavior before we ship the beta, as it's critical to get the lane + platform selection right 👍

### PR DESCRIPTION
**Pull request by minuscorp**
Possibly fixes #797 ?

### CODE REVIEW

**Comment by KrauseFx** [comment]
Did you test if it fixes #797 😉

File: app/features/project/project_controller.rb [hunk 1]
```diff
@@ -76,9 +76,10 @@ class ProjectController < AuthenticatedControllerBase
     post "#{HOME}/:project_id/save" do
       project_id = params[:project_id]
       project = user_project_with_id(project_id: project_id)
-      project.lane = params["selected_lane"]
+      project.platform = params["selected_lane"].split(" ").first
+      project.lane = params["selected_lane"].split(" ").last
       project.project_name = params["project_name"]
-
+
```
Comment (KrauseFx) [inline app/features/project/project_controller.rb]: rubocop doesn't complain about trailing white space here?

Comment (minuscorp) [reply app/features/project/project_controller.rb]: Surely, my local Rubocop integration with vscode is behaving weirdly lately. Gonna check soon!

**Comment by minuscorp** [comment]
Afaik, I couldn't test that saving the form was causing issues, so I'd appreciate feedback!

**Pull request closed without merging**